docs/generation/site-remote.yml

```diff
@@ -68,6 +68,10 @@ content:
       branches: main
       tags: [0.25.0, 0.26.0, 0.27.0, 0.28.0, 0.29.0, 0.30.0, 0.31.0, 0.32.0, 0.33.0, 0.34.0, 0.35.0, 0.36.0, 0.37.0, 0.38.0, 0.39.0, 0.40.0, 0.41.14, 0.42.50]
       start_path: servicetalk-client-api/docs
+    - url: ../../
+      branches: main
+      tags: [0.42.50]
```

Comment:
This array tells codegen what tags already have this new section. For new sections, we always start with an empty array. Otherwise, we are getting an exception:
```
Error: the start path 'servicetalk-traffic-resilience-http/docs' does not exist in /Users/ipivnitskiy/Code/servicetalk/servicetalk/ (ref: tags/0.42.50)
```
Tags are generated and this array is populated when we cut a new release.

Removed in #3098.

cc @daschl who originally commended to add the number

Reply:
Now you can see that release script automatically added 0.42.51 to that empty array and pushed docs under a new git tag: https://github.com/apple/servicetalk/commit/5d187f780036cae32a395d44abdee67b95be524d#diff-9bb3b0a6e6b4f535d65df7235e1c7c296913e49d3da216dc57757a4dd0d95ff3R73